istioctl/pkg/writer/envoy/configdump/cluster.go

```diff
@@ -73,9 +73,9 @@ func (c *ConfigWriter) PrintClusterSummary(filter ClusterFilter) error {
 				if subset == "" {
 					subset = "-"
 				}
-				fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%v\n", fqdn, port, subset, direction, cluster.ClusterDiscoveryType)
+				fmt.Fprintf(w, "%v\t%v\t%v\t%v\t%s\n", fqdn, port, subset, direction, cluster.GetType())
```

Comment:
@kyessenov made a recent change where this was changed from getType to clusterDiscoveryType. You should consult with him..